messaging/src/main/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessor.java

```diff
@@ -205,6 +205,7 @@ private void initializeTokenStore() {
             } catch (Exception e) {
                 logger.info("Error while initializing the Token Store. " +
                                     "This may simply indicate concurrent attempts to initialize.", e);
+                initializeTokenStore();
```

Comment:
Not sure if it's wise to retry this from within the transaction, and also doing that immediately, without any retry mechanism.
If we want the initialization to happen in retries, then we would need the coordinator to schedule and reschedule this action if it failed initially.

Reply:
Move the responsibility of kick-starting the tokens for a given PSEP sounds feasible to me. Benefit is that it already has an `Executor` in there, which we can proceed the internal coordination task for example.

Reply:
Changed it to keep retrying, with a retry interval, but still before the coordinator is started. Not sure that's the best approach, but it's better than the initial solution. Also not sure if we want to make the retry interval configurable, and if we want to retry only a certain times, and than throw a `RetriesExhaustedException` or something.

Reply:
Tokens are now also initialized by the coordinator.